### ISSUE TITLE
Improve README.md

### ISSUE THREAD

**Issue by keivalya** [keivalya/dungeon#1]
Need a detailed README, on how to play the game

**Comment by LucasPilla** [comment]
I may be able to help you! Can you assign this issue to me?

**Comment by ani-coder-hacker** [comment]
Can I work on this? I can document the game process.

**Comment by ani-coder-hacker** [comment]
I made a pull request, but unfortunately, it is showing that I don't have *write* access for the repository.

**Comment by keivalya** [comment]
@ani-coder-hacker 
I have updated the repository, hence your forked repo must be few commits behind mine.
Update it, and then try.

**Comment by wiredfoxterrier** [comment]
Hey @ani-coder-hacker 
I would like to update the README file and make it more appealing for first-time users. It would be great if you could assign this to me.
Thanks,
Ansh Sharma
(wiredfoxterrier)

**Comment by anshsharma-17** [comment]
Hey @keivalya 
I would like to update the README file and make it more appealing for first-time users. It would be great if you could assign this to me.
Thanks,
Ansh Sharma
(wiredfoxterrier)

**Comment by keivalya** [comment]
@wiredfoxterrier Sure, give it a try

**Comment by anshsharma-17** [comment]
@keivalya Have submitted a pull request please let me know if there are any changes to be made.

Thanks,
Ansh Sharma
(wiredfoxterrier)

**Comment by keivalya** [comment]
@wiredfoxterrier can you contact me in person?
LinkedIn or IG? Anywhere? I seem to have some issues merging your PRs.
I can help you with that.
[Keivalya](https://www.linkedin.com/in/keivalya)
or
"keivalya" my IG username

**Comment by anshsharma-17** [comment]
Yes sure my Linkedin:  https://www.linkedin.com/in/ansh-sharma-73a5051a6/ and Instagram: https://www.instagram.com/anshsharma17_/

**Comment by anshsharma-17** [comment]
@keivalya I have messaged you on IG. You could message me there.

**Comment by King-Del-ErnestO** [comment]
Is this still open???🤔

**Comment by keivalya** [comment]
@King-Del-ErnestO yes. Improvements can be made anytime!
It's open for all.
